python/hail/matrixtable.py

```diff
@@ -367,7 +367,17 @@ def global_schema(self):
 
     @property
     def col_key(self):
-        """The list of column key fields.
+        """Return a struct expression with the column keys.
```

Comment:
keys should be singular everywhere, maybe this should be something like "Struct of column key fields". Will fix in a future PR